AdminUi/src/AdminUi/ClientApp/src/app/components/quotas/identity/identity-details/identity-details.component.html

```diff
@@ -128,6 +128,7 @@ <h2 class="header-title">{{ header }}</h2>
                     <td class="mat-cell no-data" [attr.colspan]="quotasTableDisplayedColumns.length">No quotas found.</td>
                 </tr>
             </table>
+            <span id="quota-tooltip" class="visually-hidden">Tier quotas do not take effect if there is an individual quota.</span>
```

Comment:
Didn't we already have a tooltip? Why did you have to create another one?

Reply:
You just marked this conversation as resolved without any answer, and the code didn't change.

Reply:
I did change it https://github.com/nmshd/backbone/pull/518/commits/c917082586355806baf1d9f503ff4a2f7434bc6d

Reply:
<img width="422" alt="image" src="https://github.com/nmshd/backbone/assets/37671301/3151ede7-2173-4c58-9944-994e526ee225">

In the task it self, i was to implement alt-text for disabled quotas, so my idea was when there was any Tier Quota present this text would appear below every other data, explaining the idea.

Reply:
Yes it does. :)